shared/util.go

```diff
@@ -652,6 +652,14 @@ func IsTrue(value string) bool {
 	return StringInSlice(strings.ToLower(value), []string{"true", "1", "yes", "on"})
 }
 
+func IsNotBlank(value string) bool {
```

Comment:
@my5t3ry I think we can avoid needing these functions right now. As we use direct `==""` comparison elsewhere for this type of check, such as https://github.com/lxc/lxd/blob/master/lxd/device/nic_bridged.go#L118

I wouldn't want to start introducing inconsistent behaviour.